src/styles/components/Breadcrumb.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ @import "../common/colors";
+ @import "../common/fonts";
+ @import "../common/mixins";
+
+ .breadcrumb1 {
+    display: block; 
+    width: fit-content;
+    padding: 23px 39px; 
+    &__breadcrumb-item-active{
+      @include breadcrumb-item($gray-900)  
+   }
+   &__breadcrumb-item{
+      @include breadcrumb-item($blue-600)  
+   }
+ }
+
+ .breadcrumb {
+   display: flex;
+   flex-wrap: wrap;
+   padding: 0 0;
+   margin-bottom: 1rem;
+   list-style: none;
+ }
+ 
+.breadcrumb-item {
+	+ {
+		.breadcrumb-item {
+			padding-left: 0.5rem;
+			&::before {
+				float: left;
+				padding-right: 0.5rem;
+				color: $breadcrumb-separator-color;
+				content: var(--bs-breadcrumb-divider, "/");
+			}
+		}
+	}
+}
+.breadcrumb-item.active {
```

Comment:
Nest like below in SASS style to keep all styles under .breadcrumb-item
`.breadcrumb-item {
    &.active {
    }
}`

Reply:
done